src/auth/jwt.guard.ts

```diff
@@ -28,6 +28,10 @@ export class AccessTokenAuthGuard extends AuthGuard("accessToken") {
       if (error.message === "invalid token") {
         console.log(error.message);
         throw new HttpException("invalid token", 400);
+      }
+      if (error.message === "invalid signature") {
+        console.log(error.message);
+        throw new HttpException("invalid signature", 400);
```

Comment:
```suggestion
      if (error.message === "invalid signature") {
        console.log(error.message);
        throw new HttpException(error.message, 400);
```
똑같은 에러 메시지를 그대로 넘길거라면 굳이 문자열로 타이핑할 필요 없이 그대로 error.message를 넘기셔도 될 거 같아요